db/changes_view.go

```diff
@@ -78,6 +128,20 @@ func (dbc *DatabaseContext) getChangesInChannelFromView(
 			highSeq = entry.Sequence
 		}
 
+		// Close query results
+		closeErr := queryResults.Close()
+		if closeErr != nil {
+			return nil, closeErr
+		}
+
+		if queryRowCount == 0 {
+			if len(entries) > 0 {
+				break
+			}
+			base.LogTo("Cache", "    Got no rows from query for %q", channelName)
```

Comment:
Should `channelName` be redacted?

Also I think it should be labeled, since "Got no rows from query for banana" is less then confusing than "Got no rows from query for channel 'banana'"